packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -595,6 +609,17 @@ export class Combobox extends FormAssociatedCombobox {
         }
     }
 
+    /**
+     * Determines if a value update should involve emitting a change event
+     *
+     * @internal
+     */
+    private shouldEmitChange(): boolean {
```

Comment:
Not sure how I feel about this method as it may not be appropriate to use it to make the determination to emit the `change` event (like in the case of a form reset). However, this seemed preferable to duplicating the logic in two places.

Reply:
Updated as suggested.